backend/src/nodes/categories.py

```diff
@@ -18,15 +18,105 @@
 # Utility
 UTILITY = "Utility"
 
-category_order = [
-    IMAGE,
-    IMAGE_DIMENSION,
-    IMAGE_ADJUSTMENT,
-    IMAGE_FILTER,
-    IMAGE_UTILITY,
-    IMAGE_CHANNEL,
-    PYTORCH,
-    ONNX,
-    NCNN,
-    UTILITY,
+
+class Category:
+    def __init__(
+        self, name: str, description: str, icon: str, color: str, install_hint: str = ""
```

Comment:
Please type the install hint as nullable instead of using the empty string as sentinel value.